src/natcap/invest/ui/model.py

```diff
@@ -1268,7 +1265,7 @@ def _settings_saved_message():
                 model=modelname)))
 
         self.suffix = inputs.Text(
-            args_key=suffix_args_key,
+            args_key='results_suffix',
```

Comment:
Right, since `results_suffix` is standardized across InVEST there shouldn't be the option to name it something else. (just commenting to confirm my own understanding)

Reply:
Yep, there shouldn't be that option at all, and this is something that should have been addressed way back when we were building this version of the UI.  It was only fairly recently that we actually did formally establish a standard suffix.